amlb/resources.py

```diff
@@ -210,14 +210,14 @@ def _validate_task(self, task, lenient=False):
 
         conf = 'id'
         if task[conf] is None:
-            task[conf] = ("openml.org/t/{}".format(task.openml_task_id) if task['openml_task_id'] is not None
-                          else "openml.org/d/{}".format(task.openml_dataset_id) if task['openml_dataset_id'] is not None
```

Comment:
@mfeurer the deletion of this part is causing the new error.
This is to support the legacy format that we're using in all `yml` files, especially the `test.yaml` one.
I agree that ideally, this should not be here but in `openml.py` module, but right now, I don't have time to look into this, so you can just restore this to get the test wroking again.